### DISCUSSION TITLE
Where did the SeelenUI name came from?

### DISCUSSION THREAD

**Discussion by SoftwareType** [eythaann/Seelen-UI#22] [Q&A] [answered]
Specifically the word "Seelen", where did the idea of the word came from? What does it mean, if theres any?

The best I could get searching is by a surname, but haven't found the meaning from it either

**Comment by eythaann** [comment, marked as answer]
The name "Seelen" comes from the German word for "souls." Initially, the app was just a dock and was called "Seelen Weg." In this context, "Weg" means "path" or "way," so the dock was the "way" or "path" for the "souls," representing the apps you use. As the project evolved, I added more widgets like the Window Manager and the topbar, so I decided to simplify the name to just "Seelen UI."
